### ISSUE TITLE
Text spacing violation (clipping) in some assessment left nav items

### ISSUE THREAD

**Issue by dbjorge** [microsoft/accessibility-insights-web#4361]
## Describe the bug

In the web details view, the left nav menu for assessments has several items which get clipped when you apply the styles suggested by our own Text Spacing test:

![screenshot of issue with the clipped nav items circled](https://user-images.githubusercontent.com/376284/121971822-a46e5b80-cd47-11eb-933d-7591f631f3a8.png)

Additionally, the `scrollable-region-focusable` automated check gets clipped slightly even with default styling:

![screenshot showing scrollable-region-focusable clipping](https://user-images.githubusercontent.com/376284/121971908-dd0e3500-cd47-11eb-97f7-1aec7dde9d36.png)

**To Reproduce**

1. Open a web assessment against any page
2. Right click on one of the menu items and select "Inspect"
3. In the F12 developer tools panel that opens, apply the following styles:
```css
    line-height: 1.5 !important;
    letter-spacing: 0.12em !important;
    word-spacing: 0.16em !important;
```
4. Observe the results from the above screenshot

## CodePen repro example

n/a

## Expected behavior

Per our own Text Spacing requirement (emphasis mine):

"Verify that all of the following are true:
* All text responds to each change in styling.
* ***All text remains visible (no clipping)***.
* There is no overlapping text."

## Screenshots

see inline

## Context (please complete the following information)

- OS Name & Version: Windows 21H1 OS Build 19043.1052
- AI-Web Version & Environment: Production 2.27.0 and dev cb03f4912
- Browser Version: Microsoft Edge Version 91.0.864.41 (Official build) (64-bit)
- Target Page: n/a

## Are you willing to submit a PR?

Yes

## Did you search for similar existing issues?

Yes

## Additional context

Discovered during biannual compliance pass when the scrollable-region-focusable issue with default styling tripped a need-review check run as part of a self-assessment

**Comment by dbjorge** [comment]
@juchampa / @ferBonnin , we'll need design input for what you'd like to do about these cases. I'm imagining something like a combination of "enable the text in the menu items to reflow for the sake of users that want to use custom text styling, and also expand the width slightly so `scrollable-region-focusable` doesn't reflow under default styles" - does that sound reasonable?

**Comment by ferBonnin** [comment]
> @juchampa / @ferBonnin , we'll need design input for what you'd like to do about these cases. I'm imagining something like a combination of "enable the text in the menu items to reflow for the sake of users that want to use custom text styling, and also expand the width slightly so `scrollable-region-focusable` doesn't reflow under default styles" - does that sound reasonable?

this sounds reasonable, thanks @dbjorge!

**Comment by dbjorge** [comment]
Resolved by #4644
